resources/migrations/019_add_schema_column_to_table.yaml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+  - changeSet:
+      id: 19
+      author: camsaul
+      changes:
+        - addColumn:
+            tableName: metabase_table
+            columns:
+              - column:
+                  name: schema
+                  type: VARCHAR
```

Comment:
Let's back up a second before we go crazy with varchar limits. Varchars are generally stored by data structures that have 1-4 bytes for the length and a pointer to the text itself. Similar to a `std::string` in C++:

```C++
struct example_varchar {
    int16_t length; // two bytes for length
    char *c_str;     
}
```

The moral of the story is that a 12-byte string will take 13-16 bytes to store (depending on the DB), whether the column type is a `varchar(256)` or a `varchar(1024)`. The limit is just an arbitrary DB constraint and doesn't affect performance or storage requirements of strings under that limit.

So it makes sense to use limits that give us plenty of headroom just in case we end up needing them. It looks like [SQL Server has a schema name length limit of 128 chars](http://stackoverflow.com/questions/5808332/sql-server-maximum-character-length-of-object-names). [So does DB2](http://www-01.ibm.com/support/knowledgecenter/ssw_ibm_i_71/db2/rbafzlimtabs.htm). I don't want to find out down the road that some other DB has a limit of 256 and have to go write another migration to change this :-1: